### DISCUSSION TITLE
How to show wallet's assets?

### DISCUSSION THREAD

**Discussion by fiwalletsol** [trustwallet/wallet-core#1553] [Q&A]
First Question: I want to reach a wallet's assets or total balances. Is this repo able to do this? 

Second Question: Is this repo enough to create a mobile wallet application?

**Comment by optout21** [comment]
Wallet core contains functions for HD wallet, key management, address derivation, transaction building and signing.  But it does not contain any communication with blockchain nodes, which is needed for balance retrieval, transaction history, and new transaction broadcasting.
So wallet core is an important part of a wallet, but not enough in itself.

**Reply by fiwalletsol** [reply]
So sorry to bother but can you fill me in about a few things? What should I use for those other things like communication with blockchain nodes , balance retrieval, transaction history and new transaction broadcasting?
